networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/gossip/forks/versions/GossipForkSubscriptionsDeneb.java

```diff
@@ -81,11 +86,15 @@ public GossipForkSubscriptionsDeneb(
         syncCommitteeMessageOperationProcessor,
         signedBlsToExecutionChangeOperationProcessor);
 
-    this.gossipedBlockAndBlobsProcessor = gossipedBlockAndBlobsProcessor;
+    this.blobSidecarProcessor = blobSidecarProcessor;
+    this.blockAndBlobsProcessor = blockAndBlobsProcessor;
   }
 
   @Override
   protected void addGossipManagers(final ForkInfo forkInfo) {
+    // TODO: change this to super.addGossipManagers(forkInfo) when the coupled block and blobs
```

Comment:
I don't understand why we should call `super.addGossipManagers(forkInfo)` instead. The Gossip Managers are instantiated separately is each respective method.

Reply:
Because we can leverage that Deneb extends Capella and can just call super to initialize all previous forks from gossip. But because previous version of Deneb replaced the block gossip with the block_and_blobs_sidecar, we couldn't use super. Now they are separate, we can simplify it.